plugins/core/jetbrains-community/src/software/aws/toolkits/jetbrains/core/credentials/sso/pkce/ToolkitOAuthService.kt

```diff
@@ -182,15 +182,8 @@ internal class ToolkitOAuthCallbackHandler : OAuthCallbackHandlerBase() {
         return AcceptCodeHandleResult.Redirect(urlBase.addParameters(params))
     }
 
-    override fun isSupported(request: FullHttpRequest): Boolean {
-        // only handle if we're actively waiting on a redirect
-        if (!oauthService().hasPendingRequest()) {
-            return false
-        }
-
-        // only handle the /oauth/callback endpoint
-        return request.uri().trim('/').startsWith("oauth/callback")
-    }
+    // only handle the /oauth/callback endpoint
+    override fun isSupported(request: FullHttpRequest) = request.uri().trim('/').startsWith("oauth/callback")
```

Comment:
probably also want to restrict to HTTP GET

Reply:
gotcha